mobile/app/Components/AREAsList.tsx

```diff
@@ -0,0 +1,252 @@
+/**
+ * @fileoverview This file shows the list of areas
+ */
+
+import React, {useState, useEffect} from 'react';
+import {
+    View,
+    Text,
+    TouchableOpacity,
+    Image,
+    ScrollView,
+    ActivityIndicator,
+    Switch,
+    Animated,
+    RefreshControl
+} from 'react-native';
+import {useNavigation} from 'expo-router';
+import {styled} from 'nativewind';
+import colors from "../../constants/Colors";
+import actionReactionJsonDataService from '../Utils/actionReactionJsonData.serivce';
+import {AreaDetailsInterface} from "../Interfaces/areaData.interface";
+import {Ionicons} from "@expo/vector-icons";
+import BravmmModal from "./BravmmModal";
+import {eraseArea, toggleArea} from "../Utils/callApi";
+import {FadeLoading} from 'react-native-fade-loading';
+
+const StyledView = styled(View);
+const StyledText = styled(Text);
+const StyledTouchableOpacity = styled(TouchableOpacity);
+const StyledImage = styled(Image);
+const StyledScrollView = styled(ScrollView);
+
+const servicePicture: { [key: string]: string } = {
+    Discord: require("./../../assets/images/logos/Discord_logo.png"),
+    Twitch: require("./../../assets/images/logos/Twitch_logo.png"),
+    Spotify: require("./../../assets/images/logos/Spotify_logo.png"),
+    Teams: require("./../../assets/images/logos/Teams_logo.png"),
+    Gmail: require("./../../assets/images/logos/Gmail_logo.png"),
+    Outlook: require("./../../assets/images/logos/Outlook_logo.png"),
+    TrackerGG: require("./../../assets/images/logos/TrackerGG_logo.png"),
+    Onedrive: require("./../../assets/images/logos/Onedrive_logo.png"),
+    Weather: require("./../../assets/images/logos/Weather_logo.png"),
+    Timer: require("./../../assets/images/logos/Timer_logo.png"),
+};
+
+const EditArea = ({area, setNeedRefresh, setIsModalOpen}: {
+    area: AreaDetailsInterface | undefined,
+    setNeedRefresh: React.Dispatch<React.SetStateAction<boolean>>
+    setIsModalOpen: React.Dispatch<React.SetStateAction<boolean>>
+}) => {
+    if (!area) {
+        return (
+            <ActivityIndicator size="large" color={colors.light.fourthly}/>
+        )
+    }
+    const [isActivated, setIsActivated] = useState<boolean>(area.isActivated);
+
+    return (
+        <StyledView className="flex-col items-center justify-between px-8 py-10 w-full">
+            <StyledImage
+                style={{width: 150, height: 150}}
+                // @ts-ignore
+                source={servicePicture[area.serviceName]}
```

Comment:
servicePicture should not be accessed directly.
In case of unknown serviceName it will throw an error.

You should create a function to acces it.
Inside this function you should return a generic image (BRAVM logo) instead of nothing.